packages/api-explorer/src/form-components/SchemaField.jsx

```diff
@@ -1,3 +1,5 @@
+require('../../schema-form.css');
```

Comment:
Importing CSS directly in to the JSX component lets us keep our CSS import in a single place. Otherwise we'd have to import it on demand, in multiple places (i.e. once in the demo app, once in the main ReadMe repo, etc.)

Reply:
Can you also do this for https://github.com/readmeio/api-explorer/blob/master/packages/api-explorer/api-explorer.css? Maybe just require it into the top level javascript file?

As you can see, we've just done this in the past when the component has been included:

https://github.com/readmeio/api-explorer/blob/bd6a5fe395bd23aa9121463cadbed6d50385aff9/example/src/Demo.jsx#L14-L15

In main ReadMe, we're requiring it directly into our CSS bundle as well.